code/WorkInProgress/MechanicMadness.dm

```diff
@@ -156,14 +211,18 @@
 			return ..()
 		return
 	get_desc()
-		.+="[src.welded ? " It is welded shut." : ""][src.open ? " Its cover has been opened." : ""][src.anchored ? "It is [src.open || src.welded ? "also" : ""] anchored to the ground." : ""]"
+		.+="[src.welded ? " It is welded shut." : ""][src.open ? " Its cover has been opened." : ""]\
+		[src.anchored ? "It is [src.open || src.welded ? "also" : ""] anchored to the ground." : ""]"
 	housing_large // chonker
 		can_be_welded=true
 		can_be_anchored=true
-		slots=23 // wew, dont use this in-hand or equipped!
-		name="Component Cabinet"
-		desc="A rather chunky cabinet for storing up to 23 active mechanic components at once.<br>It can only be connected to external components when bolted to the floor.<br>"
+		slots=CABINET_CAPACITY // wew, dont use this in-hand or equipped!
+		name="Component Cabinet" // i tried to replace "23" below with "[CABINET_CAPACITY]", but byond \
+									 thinks it's not a constant and refuses to work with it.
```

Comment:
oh god this one too please no

Reply:
same as above, backslash extends comments by 1 line. it compiles